### PR TITLE
Update OMERO.server to 5.0.8

### PR DESCRIPTION
**Pull request by manics**
Update to 5.0.8 release
Update CentOS instructions to use PostgreSQL 9.4

Will need copying to openmicroscopy/ome-documentation

### CODE REVIEW

File: linux/setup_omero_ice35.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ set -e -u -x
 
 source settings.env
 
-SERVER=http://downloads.openmicroscopy.org/omero/5.0.6/artifacts/OMERO.server-5.0.6-ice35-b53.zip
+SERVER=http://downloads.openmicroscopy.org/omero/5.0.8/artifacts/OMERO.server-5.0.8-ice35-b60.zip
```
Comment (sbesson) [inline linux/setup_omero_ice35.sh]: Use the new `latest/omero/server-ice35.zip` URL ?  :)

**Comment by rleigh-dundee** [comment]
The changes all look reasonable to me.

File: linux/setup_omero_ice35.sh (hunk 1 above)
Comment (manics) [inline linux/setup_omero_ice35.sh]: How about in the next PR? I'll add separate 50 and 51 scripts.

Comment (sbesson) [inline linux/setup_omero_ice35.sh]: Makes sense. Do you want `latest/omero50/server-ice35.zip` redirects ?

Comment (manics) [inline linux/setup_omero_ice35.sh]: Good question, this is getting towards having stable vs develop in the scripts. Advantage of plain `omero` is the docs will always recommend installing our most recent stable version, OTOH `omero50` is more explicit about what's happening.

**Pull request merged**